src/blog/new-truffle-release-update-march/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: New Truffle Releases Update - Ganache.dev + Hardhat plugin
+hide:
+  - navigation
+---
+
+![Blog header - New Truffle Releases Update](./what-is-new-in-truffle-banner.png)
+
+**By [Kingsley Arinze](https://www.linkedin.com/in/bruzzy)**
+
+If you’ve ever wondered what RPC methods [Ganache](https://trufflesuite.com/ganache/) support or what arguments to pass in when calling these RPC methods, including the type of response to expect, then we’re happy to introduce you to [ganache.dev](http://ganache.dev), an interactive JSON-RPC documentation for Ganache.
+
+Or if you’re a Hardhat user but can not stop thinking of what could be if you had a tool like [Truffle Dashboard](https://trufflesuite.com/blog/introducing-truffle-dashboard/), that plugs in seamlessly to your existing workflow, allowing you to interact with the blockchain and deploy contracts without exposing your secret keys, then look no further as we’ve created a plugin that extends Truffle Dashboard to do just that.
+
+In this product release update, we'll delve into [ganache.dev](http://ganache.dev) and [Truffle Dashboard Hardhat Plugin](https://www.npmjs.com/package/@truffle/dashboard-hardhat-plugin), what they are, and how to use them to improve your development experience.
+
+## ganache.dev, An interactive JSON-RPC documentation for Ganache
+
+We’re happy to share with the world this project we’ve been hard at work on for the past year or so, [ganache.dev](http://ganache.dev). [ganache.dev](http://ganache.dev) is an interactive JSON-RPC documentation for Ganache that allows you to quickly try out various RPC methods on the browser, without having to run a Ganache instance.
+
+It contains code samples for each RPC method and all code samples are powered by [Ganache in the Browser](https://github.com/trufflesuite/ganache/#browser-use) and demonstrate using Ganache programmatically as an [EIP-1193 provider](https://github.com/trufflesuite/ganache/#as-an-eip-1193-provider-only).
```

Comment:
Another suggestion...replace "It contains code samples for each RPC method and all code samples are powered by Ganache in the Browser and demonstrate using Ganache programmatically as an EIP-1193 provider." with something like "It contains code samples for each RPC method and is powered by Ganache in the Browser, demonstrating using Ganache programmatically as an EIP-1193 provider."